### PR TITLE
feat: add Helm chart

### LINKED ISSUE #68: Deploy the project in their environments

### PR DESCRIPTION
**Pull request by tensor5**
Closes #68.

### CODE REVIEW

File: charts/developers-italia-api/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "v0.1.1-pre"
+
```
Comment (bfabio) [inline charts/developers-italia-api/Chart.yaml:25]: Can we leave `appVersion` out, so we don't have to bump it every time?

We usually do and Keel should already take care of upgrading.

File: charts/developers-italia-api/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+| ingress.className | string | `""` |  |
+| ingress.enabled | bool | `false` |  |
+| ingress.hosts[0].host | string | `"chart-example.local"` |  |
+| ingress.hosts[0].paths[0].path | string | `"/"` |  |
+| ingress.hosts[0].paths[0].pathType | string | `"ImplementationSpecific"` |  |
+| ingress.tls | list | `[]` |  |
+| maxRequests | int | `nil` | Max number of requests. |
+| nameOverride | string | `""` |  |
+| nodeSelector | object | `{}` |  |
+| pasetoKey | string | `""` | Paseto Key. |
```
Comment (bfabio) [inline charts/developers-italia-api/README.md:31]:
```suggestion
| pasetoKey | string | `""` | Base64 encoded Paseto Key. |
```

**Review by bfabio — COMMENTED** [review]
Great! Can we add Keel for upgrades?

**Comment by bfabio** [comment]
We also probably want to create a new `developers-italia-api` in https://github.com/teamdigitale/devita-infra-kubernetes containing just the README with installation instructions.

File: charts/developers-italia-api/Chart.yaml (hunk 1 above)
Comment (tensor5) [reply charts/developers-italia-api/Chart.yaml:25]: We can configure Keel directly in the `values.yml` override that will be in https://github.com/teamdigitale/devita-infra-kubernetes (see https://keel.sh/docs/#helm-example), and release this chart to the GitHub registry with appVersion tied to the latest stable Docker image.

Comment (bfabio) [reply charts/developers-italia-api/Chart.yaml:25]: @tensor5 sounds great :+1:

**Review by bfabio — APPROVED** [review]

**Pull request merged**